templates/squareup-javapoet/cgmanifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Registrations": [
+    {
+      "Component": {
+        "Type": "Maven",
+        "Maven": {
+          "ArtifactId": "okhttp",
```

Comment:
JavaPoet instead of OkHttp

Reply:
not only that.
groupId was wrong and version too
fixed